packages/components/src/components/tag/tag.css

```diff
@@ -92,7 +92,7 @@
   box-shadow: var(--box-shadow-button-dismissable-focus);
 }
 .tag--dismissable button:hover {
-  background: var(--background-button-dismissable-hover);
+  background: var(--scl-color-white);
```

Comment:
Did you consider setting `--background-button-dismissable-hover` in the root to `var(--scl-color-white)` instead of leaving it empty?

Reply:
Yes that make definitely sense. However, I actually did not notice it at all in the root. Otherwise I would have changed it there. Thanks 😁

Reply:
Yes that make definitely sense. However, I actually did not notice it at all in the root. Otherwise I would have changed it there. Thanks 😁